### PR TITLE
AP_HAL_ChibiOS: added PixPilot-C3

### PR DESCRIPTION
**Pull request by Cedric0489**

### CODE REVIEW

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
your wiki page submission (thanks for that) states that the buzzer is integrated on board....have you tested to make sure buzzer does not cause vibration of IMUs?? this can be a serious issue....

**Comment by Cedric0489** [comment]
> your wiki page submission (thanks for that) states that the buzzer is integrated on board....have you tested to make sure buzzer does not cause vibration of IMUs?? this can be a serious issue....

We have tested the vibration caused by the buzzer and its impact on imus is negligible.

**Review by Hwurzburg — APPROVED** [review]
still have a conflict...are you finished testing and ready for this to be merged?

**Comment by Cedric0489** [comment]
> still have a conflict...are you finished testing and ready for this to be merged?

Yes, we are done testing and ready to merge.

File: Tools/AP_Bootloader/board_types.txt [hunk 1]
```diff
@@ -248,10 +248,12 @@ AP_HW_PixFlamingoH743V               1133
 
 AP_HW_AR-F407SmartBat                1134
 AP_HW_SPEEDYBEEF4MINI                1135
+<<<<<<< HEAD
```
Comment (tridge) [inline Tools/AP_Bootloader/board_types.txt:251]: this is a committed merge conflict, needs fixing

**Review by tridge — CHANGES_REQUESTED** [review]

File: Tools/AP_Bootloader/board_types.txt (hunk 1 above)
Comment (Hwurzburg) [reply Tools/AP_Bootloader/board_types.txt:251]: sorry..fixed

**Review by tridge — APPROVED** [review]

**Pull request merged**